pages/en/network-transition-faq.mdx

```diff
@@ -0,0 +1,236 @@
+
+The higher the curation signal, the more attractive a subgraph is to Indexers, as there is a linear correlation between higher signal and higher indexing rewards. Without curation, there is no incentive for Indexers to pick up a subgraph.
+
+## What happens to the GRT a subgraph developer uses for curation signal?
+
+If you are the first to signal a subgraph, your GRT signaled amount will not go down. GRT used for curation can be removed later. Also, Curators get 10% of all query fees taken in by Indexers.
+
+## What improvements are coming to the curation process?
+
+Short term, the initial curation model on Arbitrum will provide principle-protection to curation signal. Longer term, the core devs will prioritize offering developers the capacity to rent curation signal, opening up a more predictable pricing experience while still ensuring subgraphs are sufficiently indexed.
+
+## How do I switch the subgraph API in the front-end?
+
+After at least one Indexer has fully indexed a subgraph, a user can query the decentralized network.
+
+In order to retrieve the query URL for your subgraph, you can copy/paste it by clicking on the symbol next to the query URL. You will see something like this:
+
+`https://gateway.thegraph.com/api/[api-key]/subgraphs/id/S9ihna8D733WTEShJ1KctSTCvY1VJ7gdVwhUujq4Ejo`
+
+Simply replace [api-key] with an API key generated in the Subgraph Studio [API Key section](https://thegraph.com/studio/apikeys/).
+
+## How much do queries cost?
+
+The average query cost within the network varies. For the month of September 2022, the average price per query fee cost ranged from $0.00012 - $0.00020.
+
+## How can I find out how much volume my subgraph has and how much it will cost?
+
+Hosted service volume data is not public. Please reach out to get volume and cost estimates [here](mailto:migration@thegraph.foundation).
+
+## How does the gateway work? Is it fully decentralized?
+
+The gateway process queries so Indexers can serve dapps. The gateways are in an intermediate phase that is being progressively decentralized. More on this soon.
+
+# Using The Network FAQs
+
+## Is there a cost to upgrade my subgraph?
+
+Yes, it is 1% of curation signaled. The 1% is split evenly between Curators (0.5%) and subgraph developers (0.5%). So, for every 10K GRT signaled, it costs subgraph developers 50 GRT to upgrade.
+
+## How do I speed up sync time?
+
+Minimize the use of smart contract calls within the subgraph. Accessing a smart contract state requires an eth_call to the RPC, which slows down sync times.
+
+## Is there multisig support in Subgraph Studio as I migrate?
+
+Yes, multisig support has recently been added. You can find more information [here](https://thegraph.com/docs/studio/multisig).
+
+## What are the contract addresses for GRT on Ethereum and Polygon?
+
+Ethereum: 0xc944E90C64B2c07662A292be6244BDf05Cda44a7
+
+Polygon: 0x5fe2B58c013d7601147DcdD68C143A77499f5531
```

Comment:
```suggestion
- Ethereum: `0xc944E90C64B2c07662A292be6244BDf05Cda44a7`
- Polygon: `0x5fe2B58c013d7601147DcdD68C143A77499f5531`
```

Reply:
Had to commit this myself

Reply:
Thanks! This was a nitpick, but I think it looks better like this.